src/transformers/modeling_bert.py

```diff
@@ -1394,12 +1394,13 @@ def forward(
 
         outputs = (logits,) + outputs[2:]  # add hidden states and attention if they are here
         if labels is not None:
-            loss_fct = CrossEntropyLoss()
+            ignore_index = -100
+            loss_fct = CrossEntropyLoss(ignore_index=ignore_index)
```

Comment:
Is there a reason for explicitly specifying the ignored index? Up to now, our philosophy was to let the default be PyTorch's default.
Explicitly specifying it means that if PyTorch ever changes the default, our losses will be incoherent with PyTorch's official losses.

Reply:
Cool, I switched to that.